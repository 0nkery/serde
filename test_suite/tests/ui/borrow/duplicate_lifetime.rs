#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
struct Test<'a> {
    #[serde(borrow = "'a + 'a")]
    s: &'a str,
}

fn main() {}
