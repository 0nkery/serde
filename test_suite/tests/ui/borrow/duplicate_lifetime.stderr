error: duplicate borrowed lifetime `'a`
  --> $DIR/duplicate_lifetime.rs:12:10
   |
12 | #[derive(Deserialize)]
   |          ^^^^^^^^^^^

error: aborting due to previous error

